app/controllers/admin/appropriate_bodies/users_controller.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+module Admin
+  module AppropriateBodies
+    class UsersController < Admin::BaseController
+      skip_after_action :verify_authorized
```

Comment:
should `verify_authorized` and `verify_policy_scoped` be scoped to only/except index 🤔 ?

Reply:
@cwrw 
Corrected the pundit policies 👍